MUICore/codegen.sh

```diff
@@ -1 +1,2 @@
-python -m grpc_tools.protoc -I../MUI/src/main/proto --python_out=muicore --grpc_python_out=muicore ../MUI/src/main/proto/MUICore.proto
+python -m grpc_tools.protoc -I. --python_out=muicore --grpc_python_out=muicore MUICore.proto
+sed -i '5s/./from . &/' muicore/MUICore_pb2_grpc.py
```

Comment:
unfortunately, there appears to be no way to have grpc_tools generate a python file that uses relative imports/recognise parent module, despite protoc itself implementing the feature recently. see: https://github.com/grpc/grpc/issues/9575 among other issues

Reply:
Sad, but I guess it is what it is 😞